projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/order-history.e2e.cy.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { viewportContext } from '../../helpers/viewport-context';
+
+describe('Order History Page accessibility', { testIsolation: false }, () => {
+  viewportContext(['mobile', 'desktop'], () => {
+    before(() => {
+      cy.a11yContinuumSetup();
+      cy.login('test-user-with-orders@sap.cx.com', 'pw4all');
+      cy.visit('/my-account/orders');
+    });
+
+    context('Order list', () => {
+      it('Page loaded', () => {
```

Comment:
```suggestion
      it('Order list', () => {
```

Is there a need for the extra context here?